src/utilities/conversion.js

```diff
@@ -24,15 +24,13 @@ import p5 from '../core/main';
  * let str = '20';
  * let diameter = float(str);
  * ellipse(width / 2, height / 2, diameter, diameter);
+ * describe(`20×20 white ellipse in the center of the canvas`);
```

Comment:
can you change text to 20 by 20. Screen readers would read "20x20" as twenty ex twenty

Reply:
Fixed in 9c0e4fb7